### PR TITLE
Docs/images

### PR DESCRIPTION
**Pull request by linonetwo**
Fix most of errors when building the doc site. I believe this will take effect after you merge rel2.0.0 to the master.

You can try doc locally using `npm run docs:dev` (`npm i` first)

### CODE REVIEW

**Comment by linonetwo** [comment]
截屏2023-12-21 14 13 15

File: docs/bpmn-changes.md [hunk 1]
```diff
@@ -23,45 +31,47 @@ In Process definition (.bpmn file), use `implementation` attribute to define nam
...
 		this.log('testing from the inside: '+this.data.loopKey);
 	</bpmn2:conditionExpression>
 
 
 ```
+
 ### Input Fields
-```ts
+
+````ts
```
Comment (Zacharia2) [inline docs/bpmn-changes.md:61]: 这个地方是多了一个嘛？

Comment (linonetwo) [reply docs/bpmn-changes.md:61]: 应该是，批量替换的

**Comment by linonetwo** [comment]
@ralphhanna Hi, don't forget this. While it won't take effect until being master branch.

**Pull request merged**